### PR TITLE
some build improvements

### PR DESCRIPTION
**Pull request by brunobar79**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "detox:android": "detox build -c android.emu.debug && detox test -c android.emu.debug --loglevel verbose",
     "detox:android:release": "detox build -c android.emu.release && detox test -c android.emu.release",
     "start": "react-native start",
-    "start:clean": "watchman watch-del-all && rm -rf $TMPDIR/react-* && react-native start -- --reset-cache",
+    "start:clean": "watchman watch-del-all && rm -rf $TMPDIR/react-* && rm -rf $TMPDIR/metro-* &&  rm -rf $TMPDIR/haste-map-* &&  react-native start -- --reset-cache",
```
Comment (osdnk) [inline package.json:15]: Outrageous double space

**Review by osdnk — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (brunobar79) [reply package.json:15]: I wish I had your eye sight homie!

Comment (jinchung) [reply package.json:15]: blocker

Comment (brunobar79) [reply package.json]: unblocked!  🤓

**Review by osdnk — APPROVED** [review]

**Review by jinchung — APPROVED** [review]

**Pull request merged**